tests/test_timefuncs.py

```diff
@@ -750,6 +750,12 @@ def test_some_offsets() -> None:
     )
 
 
+def test_timestampseries_offset() -> None:
+    """Test that adding an offset to a timestamp series works."""
+    vv = pd.bdate_range("2024-09-01", "2024-09-10")
+    vv + pd.tseries.offsets.YearEnd(0)
```

Comment:
Can you please assert the type of the return value (`check(assert_type())`)?

Reply:
Sorry I was to address two issues at once and put in a half-baked test.
Should be fixed now.